project/Build.scala

```diff
@@ -1230,6 +1159,10 @@ object Build {
   val generateScalaDocumentation = inputKey[Unit]("Generate documentation for dotty lib")
   val generateTestcasesDocumentation  = taskKey[Unit]("Generate documentation for testcases, usefull for debugging tests")
   lazy val `scaladoc` = project.in(file("scaladoc")).asScaladoc
+  lazy val `scaladoc-nonBootstrapped` = project.in(file("scaladoc")).scaladocBasic(NonBootstrapped).settings(
+    sources.in(Test) := Nil
```

Comment:
Can the tests not be run with the nonbootstrapped project for some reason? If so, that's worth explaining in a comment.

Reply:
I will add a comment.